pod-operation/src/utils/gpio.rs

```diff
@@ -0,0 +1,13 @@
+#[non_exhaustive]
+pub struct GpioPins;
+
+// Not discriminant but removes need to cast
+impl GpioPins {
+	const _RESERVED_I2C_SDA: u8 = 2;
+	const _RESERVED_I2C_SCL: u8 = 3;
```

Comment:
Is there a reason why you left these as private since some of our peripherals will be using these pins?